countrycodepicker/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="search_hint">Buscar &#8230;</string>
+  <string name="no_result_found">Nenhum resultado foi encontrado</string>
+  <string name="select_country">Selecionar país</string>
```

Comment:
I personally think these are better than what we added in https://github.com/joielechong/CountryCodePicker/pull/92/commits/faeacd58100e68df92fb49ee7aa0b4671c2d4482
But then, I don't speak Portuguese, just a tiny bit of Spanish 😉